lib/pages/chat/input_bar/input_bar_shortcut.dart

```diff
@@ -37,6 +40,10 @@ class InputBarShortcuts extends StatelessWidget {
         ): () {
           _debouncer.run(() {
             controller?.addNewLine();
+            _debouncer.run(() {
+              scrollController
```

Comment:
why two debouncers ?

Reply:
sr, i try with debouncer, but not work, so i use Timer. But the general idea is that we need an interval so that the `maxScrollExtent` can be updated correctly when make a new line everytime